config/runtime.exs

```diff
@@ -1,75 +1,19 @@
 import Config
 
+alias Jellyfish.ConfigReader
+
 # config/runtime.exs is executed for all environments, including
 # during releases. It is executed after compilation and before the
 # system starts, so it is typically used to load production configuration
 # and secrets from environment variables or elsewhere. Do not define
 # any compile-time configuration in here, as it won't be applied.
 # The block below contains prod specific runtime configuration.
-defmodule ConfigParser do
-  def parse_integrated_turn_ip(addr) do
-    addr = addr |> to_charlist()
-
-    case :inet.parse_address(addr) do
-      {:ok, parsed_ip} ->
-        parsed_ip
-
-      _error ->
-        with {:ok, parsed_ip} <- :inet.getaddr(addr, :inet) do
-          parsed_ip
-        else
-          _error ->
-            raise("""
-            Bad integrated TURN address. Expected IPv4 or a valid hostname, got: \
-            #{inspect(addr)}
-            """)
-        end
-    end
-  end
-
-  def parse_integrated_turn_port_range(range) do
-    with [str1, str2] <- String.split(range, "-"),
-         from when from in 0..65_535 <- String.to_integer(str1),
-         to when to in from..65_535 and from <= to <- String.to_integer(str2) do
-      {from, to}
-    else
-      _else ->
-        raise("""
-        Bad INTEGRATED_TURN_PORT_RANGE environment variable value. Expected "from-to", where `from` and `to` \
-        are numbers between 0 and 65535 and `from` is not bigger than `to`, got: \
-        #{inspect(range)}
-        """)
-    end
-  end
-
-  def parse_port_number(nil, _var_name), do: nil
-
-  def parse_port_number(var_value, var_name) do
-    with {port, _sufix} when port in 1..65535 <- Integer.parse(var_value) do
-      port
-    else
-      _var ->
-        raise(
-          "Bad #{var_name} environment variable value. Expected valid port number, got: #{inspect(var_value)}"
-        )
-    end
-  end
-
-  def get_env!(env_key) do
-    case System.get_env(env_key) do
-      nil -> raise("Environmental variable #{env_key} was not set properly")
-      env_val -> env_val
-    end
-  end
-end
+config :ex_dtls, impl: :nif
+config :opentelemetry, traces_exporter: :none
 
-hosts =
-  System.get_env("NODES", "")
-  |> String.split(" ")
-  |> Enum.reject(&(&1 == ""))
-  |> Enum.map(&String.to_atom(&1))
+hosts = ConfigReader.read_nodes("NODES")
 
-unless Enum.empty?(hosts) do
+if hosts do
```

Comment:
`hosts` may be a nonempty list, `[]` or `nil` here -- 1) and 3) are obviously fine, but could you confirm that setting `NODES=""` doesn't break anything?